readme.md

```diff
@@ -24,3 +24,28 @@ from tcii18n import Translator
 translator = Translator('my_file.csv')
 translator.translate('My string')
 ```
+
+For using this project into a Flask application.
```

Comment:
Instead of this block here, I'd rather read the code sample first and then explaining the code under the sample:

(code samples that you wrote)
…

Where:

- `get_translations_file()` is a function that returns the appropriate translation file path,
- `flask_methods(app, get_translation_file)` initializes the template processor with `app` as your Flask instance and `get_translation_file` is the reference to the callable method.